secrets/org.osbuild.rhsm

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+import json
+import os
+import sys
+import glob
```

Comment:
```suggestion
import glob
import json
import os
import sys
```

Alphabetical order ;) (See, I can spot other things than typos!)

Reply:
The order is fixed in sources/org.osbuild.files